### ISSUE TITLE
How to start!

### ISSUE THREAD

**Issue by hasan3ysf** [hiddentao/squel#202]
Dears,
I did not get how to start, where and how can I tell the Squel about my database name, user and password, i.e. where can I define my database parameters!

Thanks

**Comment by hiddentao** [comment]
Squel is just a query string builder. It won't actually connect to your database engine. Are you using node.js?

**Comment by hasan3ysf** [comment]
Yes, I'm currently using tedious http://pekim.github.io/tedious/ to interact with my MSSQL database.
